pandas/tests/frame/methods/test_downcast.py

```diff
@@ -0,0 +1,10 @@
+from pandas import DataFrame
+import pandas._testing as tm
+
+
+class TestDowncast:
+    def test_downcast(self):
```

Comment:
parametrize over frame_or_series?

Reply:
I think this makes it more complicated

Reply:
the alternative is to implement an analogous test in the series tests

Reply:
just do

```
if frame_or_series is Series:
    obj = obj["A"]
    expected = expected["A"]

result = ...
```
?

Reply:
I added a series test already, should have commented. Are you ok with that?